plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduRecordCursor.java

```diff
@@ -23,10 +23,10 @@
 import org.apache.kudu.client.KeyEncoderAccessor;
 import org.apache.kudu.client.KuduException;
 import org.apache.kudu.client.KuduScanner;
+import org.apache.kudu.client.KuduScannerIterator;
```

Comment:
Similarly can we rename the commit message as 

```
Use KuduScannerIterator instead of RowResultIterator
```
```
KuduScannerIterator sends keep alive requests to ensure Kudu scanner does not time out
```

Can be a part of commit description